blog/models.py

```diff
@@ -81,6 +82,11 @@ def image(self):
         if len(self.images):
             return self.images[0]
 
+    def delete(self):
```

Comment:
I've try the code before and after the changes and the code works good. The article is correctly remove and the picture's folders too